backend/src/test/java/com/staccato/auth/service/AuthServiceTest.java

```diff
@@ -0,0 +1,49 @@
+package com.staccato.auth.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.staccato.ServiceSliceTest;
+import com.staccato.auth.service.dto.request.LoginRequest;
+import com.staccato.exception.StaccatoException;
+import com.staccato.member.domain.Member;
+import com.staccato.member.repository.MemberRepository;
+
+class AuthServiceTest extends ServiceSliceTest {
+    @Autowired
+    private AuthService authService;
+    @Autowired
+    private MemberRepository memberRepository;
+
+    @DisplayName("입력받은 닉네임으로 멤버를 저장하고, 토큰을 생성한다.")
+    @Test
+    void login() {
+        // given
+        String nickname = "staccato";
+        LoginRequest loginRequest = new LoginRequest(nickname);
+
+        // when
+        authService.login(loginRequest);
+
+        // then
+        assertThat(memberRepository.findAll()).hasSize(1);
+    }
+
+    @DisplayName("입력받은 닉네임이 이미 존재하는 닉네임인 경우 예외가 발생한다.")
+    @Test
+    void cannotLoginByDuplicated() {
+        // given
+        String nickname = "staccato";
+        memberRepository.save(Member.builder().nickname(nickname).build());
+        LoginRequest loginRequest = new LoginRequest(nickname);
```

Comment:
```suggestion
        // given
        String nickname = "staccato";
        LoginRequest loginRequest = new LoginRequest(nickname);
        authService.login(loginRequest));
```

이렇게 하지 않은 이유가 있을까요? 제가 놓치고 있는 부분이 있다면 알려주세요.

Reply:
이미 저장되어있던 member를 만들고자 이렇게 했습니다.
제가 검증하고자 하는 메서드 `authService.login(loginRequest)`를 사용하는 것도 방법이겠지만, 또는 테스트가 실패할 이유가 "중복되는 닉네임" 뿐만 아니라 `authService.login()`의 오류가 될 수 있습니다.
그래서 테스트하고자 하는 행위를 분명히 하고자 사전 데이터는 repository를 사용했습니다.